client/service/src/lib.rs

```diff
@@ -115,8 +116,14 @@ impl RpcHandlers {
 			.map(|res| res.expect("this should never fail"))
 			.boxed()
 	}
+
+	/// Provides access to the underlying `MetaIoHandler`
+	pub fn io_handler(&self) -> Arc<jsonrpc_core::MetaIoHandler<sc_rpc::Metadata>> {
+		self.0.clone()
+	}
```

Comment:
Do you absolutely need this function, or can you add the `MetaIoHandler` methods directly to `RpcHandlers`? If so, that would be cleaner and more abstract IMO.

Reply:
Ok so the use case for this is to allow end users (and me :innocent: ) use the `jsonrpc-core-client` [local transport](https://docs.rs/jsonrpc-client-transports/14.2.1/jsonrpc_client_transports/transports/local/fn.connect.html), which takes a `Deref<Target=MetaIoHandler> + 'static`. Hence the need to have the `MetaIoHandler` behind an `Arc`.

Reply:
Ok, fair enough : )